lib/standards.h

```diff
@@ -107,9 +125,28 @@ struct Standards {
             return "c++17";
         case CPP20:
             return "c++20";
+        case CPP23:
+            return "c++23";
         }
         return "";
     }
+    static cppstd_t getCPP(const std::string &std) {
+        if (std == "c++03") {
+            return Standards::CPP03;
+        } else if (std == "c++11") {
```

Comment:
Nit; I don't have a strong opinion but would not write these redundant `else`.

Reply:
So does clang-tidy...too haste copy and paste.